helpers/uuid.go

```diff
@@ -0,0 +1,49 @@
+package helpers
+
+import (
+	"fmt"
+
+	"github.com/satori/go.uuid"
+)
+
+// GenerateUUIDv4AsString returns random generated canonical string representation of UUID:
+// xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx.
+func GenerateUUIDv4AsString() string {
+	return uuid.NewV4().String()
+}
+
+// VerifyUUIDv4String verification UUID as string for conformity with version 4
+func VerifyUUIDv4String(src string) bool {
```

Comment:
for what you're using versioning? we use only v4

Reply:
yes

Reply:
"yes" what?

Reply:
always see that we are working with version 4

Reply:
if we use v4 only, why not VerifyUUID?

Reply:
you know it. eliminate uncertainties.